### PR TITLE
feat: Add IdentityModel

### PR DESCRIPTION
**Pull request by zhenlei520**

### CODE REVIEW

File: Masa.Contrib.sln [hunk 1]
```diff
@@ -223,10 +223,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Masa.Contrib.BasicAbility.A
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Masa.Contrib.Data.IdGenerator.Snowflake.Distributed.Redis", "src\Data\IdGenerator\Masa.Contrib.Data.IdGenerator.Snowflake.Distributed.Redis\Masa.Contrib.Data.IdGenerator.Snowflake.Distributed.Redis.csproj", "{DC50078D-D706-4CB9-A301-F47CB3F46007}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Masa.BuildingBlocks.IdentityModel", "src\BuildingBlocks\MASA.BuildingBlocks\src\Identity\Masa.BuildingBlocks.IdentityModel\Masa.BuildingBlocks.IdentityModel.csproj", "{2FD02408-DF1D-48E0-94F2-9910E76D8B6E}"
```
Comment (doddgu) [inline Masa.Contrib.sln:226]: `Masa.BuildingBlocks.Identity.IdentityModel`

File: src/Identity/Masa.Contrib.IdentityModel/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Install-Package Masa.Contrib.Identity
+````
+
+1. Modify `Program.cs`
+
+```` C#
+builder.Services.AddMasaIdentity();
+````
+
+2. Obtain user information
```
Comment (doddgu) [inline src/Identity/Masa.Contrib.IdentityModel/README.md:17]: Obtain -> Get

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Identity/Masa.Contrib.IdentityModel/README.md (hunk 2 above)
Comment (zhenlei520) [reply src/Identity/Masa.Contrib.IdentityModel/README.md]: processed

File: Masa.Contrib.sln (hunk 1 above)
Comment (zhenlei520) [reply Masa.Contrib.sln]: processed

File: src/Data/Masa.Contrib.Data.Contracts.EF/DataFiltering/SoftDeleteSaveChangesFilter.cs [hunk 3]
```diff
@@ -1,6 +1,8 @@
 // Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
+using Masa.BuildingBlocks.Identity.IdentityModel;
```
Comment (doddgu) [inline src/Data/Masa.Contrib.Data.Contracts.EF/DataFiltering/SoftDeleteSaveChangesFilter.cs:4]: global

File: src/Data/Masa.Contrib.Data.EntityFrameworkCore/Filters/SaveChangeFilter.cs [hunk 4]
```diff
@@ -1,6 +1,8 @@
 ﻿// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
+using Masa.BuildingBlocks.Identity.IdentityModel;
```
Comment (doddgu) [inline src/Data/Masa.Contrib.Data.EntityFrameworkCore/Filters/SaveChangeFilter.cs:4]: global

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Data/Masa.Contrib.Data.Contracts.EF/DataFiltering/SoftDeleteSaveChangesFilter.cs (hunk 3 above)
Comment (zhenlei520) [reply src/Data/Masa.Contrib.Data.Contracts.EF/DataFiltering/SoftDeleteSaveChangesFilter.cs]: processed

File: src/Data/Masa.Contrib.Data.EntityFrameworkCore/Filters/SaveChangeFilter.cs (hunk 4 above)
Comment (zhenlei520) [reply src/Data/Masa.Contrib.Data.EntityFrameworkCore/Filters/SaveChangeFilter.cs]: processed

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**